set.mm

```diff
@@ -633125,20 +633383,40 @@ fixed reference functional determined by this vector (corresponding to
       NVROPWIQVCVEABCDFGHIVRPWIVFABCDEFGHIJKMNVROPWIQVGVH $.
   $}
 
+
   ${
-    $d N k $.  $d k ph $.
-    ineq2pow2tnd.1 $e |- ( ph -> N e. NN0 ) $.
-    $( Inequality of 2^{2n} (Contributed by metakunt, 29-Apr-2024.) $)
-    ineq2pow2tnd $p |- ( ph -> ( 2 ^ ( 2 x. N ) ) <_
-     ( ( ( 2 x. N ) + 1 ) x. ( ( 2 x. N ) _C N ) ) ) $=
-      ( vk c2 cmul co cc0 cbc csu cle cn0 wcel wceq syl wa adantr bccl nn0red
-      cz cexp cfz c1 caddc cv a1i nn0mulcld binom11 fzfid elfzelz adantl jca cr
-      2nn0 nn0zd syl2anc wbr bcmax syl2an fsumle eqbrtrd chash cfv cc fsumconst
-      cfn nn0cnd hashfz0 oveq1d eqtrd breqtrd ) AEEBFGZUAGZHVLUBGZVLBIGZDJZVLUC
-      UDGZVOFGZKAVMVNVLDUEZIGZDJZVPKAVLLMZVMWANAEBELMAUNUFCUGZDVLUHOAVNVTVODAHV
-      LUIZAVSVNMZPZVTWFWBVSTMZPVTLMWFWBWGAWBWEWCQWEWGAVSHVLUJZUKULVSVLROSAVOUMM
-      WEAVOAWBBTMVOLMWCABCUOBVLRUPZSQABLMWGVTVOKUQWECWHVSBURUSUTVAAVPVNVBVCZVOF
-      GZVRAVNVFMVOVDMVPWKNWDAVOWIVGVNVODVEUPAWJVQVOFAWBWJVQNWCVLVHOVIVJVK $.
+    resopunitintvd.1 $e |- ( ph -> ( x e. CC |-> A ) e. ( CC -cn-> CC ) ) $.
+    $( Restict continuous function on open unit interval.  (Contributed by
+       metakunt, 12-May-2024.) $)
+    resopunitintvd $p |- ( ph -> ( x e. ( 0 (,) 1 ) |-> A )
+     e. ( ( 0 (,) 1 ) -cn-> CC ) ) $=
+      ( cc0 c1 cioo co cmpt cc cres ccncf wceq cr ioossre ax-resscn sstri ax-mp
```

Comment:
I expected ioosscn here.

Reply:
Yeah good point. I kinda didn't want to move that many theorems to main, so I've replicated it, at a later stage I've found out I must move it to main due to another theorem that used it.

Otherwise it was very likely that I'd just create 50 deduction variants of theorems in main. In hindsight it would be better for us all since deduction theorems are just superior to use, but that would blow the scope of the PR.